ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx

```diff
@@ -1,4 +1,3 @@
-/** @jsxRuntime classic */
```

Comment:
The babel config should allow us to not need these pragma frags for each component that uses Emotion CSS.